src/utils/subscriptions.js

```diff
@@ -2,7 +2,7 @@ const axios = require('axios');
 const config = require('config');
 const { v4: uuidv4 } = require('uuid');
 const db = require('../storage/DataAccess');
-const fhirServerConfig = config.get('fhirServerConfig').resolve();
+const fhirServerConfig = config.fhirServerConfig;
```

Comment:
```
const fhirServerConfig = config.get('fhirServerConfig').resolve();
                                                        ^

TypeError: config.get(...).resolve is not a function
    at Object.<anonymous> (/Users/daniellee/Projects/subscription-proxy/src/utils/subscriptions.js:5:57)
```

I was getting the error above when I tried to start the server. I'm unsure of why but maybe because it's being resolved in `index.js`? Do you get the error above if you change the line back?

Reply:
I do get that error, but I don't master (I added some code to make sure sendNotification is called). Another interesting thing to note is that when using `config.fhirServerConfig`I get the error 
```
/Users/blangley/Documents/repos/medmorph/subscription-proxy/src/utils/subscriptions.js:33
          reference: `${fhirServerConfig.auth.resourceServer}/Subscription/${subscription.id}`,
                                              ^

TypeError: Cannot read property 'resourceServer' of undefined
```

Reply:
hmm so I changed it locally and it seems to work.
```
let fhirServerConfig = config.get('fhirServerConfig');
if (fhirServerConfig.resolve) fhirServerConfig = fhirServerConfig.resolve();
```
It seems like it needs to be resolved once? I don't really know how/why the config works like this.

Reply:
Should we add this check everywhere? Or can we assume it will always be resolved in `index.js` first? This seems to have fixed the weird anonymous error I was seeing though

Reply:
The `fhirClientConfig` doesn't seem to need to be resolved so I haven't run into an issue with that.

Reply:
I'm good with the rest of the code, but I'd like to better understand why this changed between master and here. Let's take a little more time to try to figure this out today. (I especially don't want the next PR to revert it if for whatever reason it breaks again)

Reply:
Sure I will take a look at this today. In `master` `sendNotification` wasn't actually being called anywhere so it wasn't imported. That would be my guess why we didn't run into this issue earlier.

Reply:
I changed both `index.js` and `subscriptions.js` to `const fhirServerConfig = config.get('fhirServerConfig');` and it seems to be working... I'm unsure why it no longer needs to be resolved... @blangley28 @dehall Do you guys still see the error?

Reply:
Working on my end... couldn't even see anything about resolve() on the config docs so I think this solution is the way to go

Reply:
Ah sorry I tested this out then forgot about it until this comment. Yes the latest is working for me. I would like to understand what the difference it but it's not worth a ton of time so I'll merge it now and we'll deal with any problems if they arise